hellohilt/src/main/java/com/airbnb/mvrx/hellohilt/HelloRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.airbnb.mvrx.hellohilt
+
+import io.reactivex.Observable
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+class HelloRepository @Inject constructor() {
+
+    fun sayHello(): Observable<String> {
+        return Observable
+            .just("Hello, world!")
+            .delay(2, TimeUnit.SECONDS)
+    }
```

Comment:
Can you make this coroutines since most new things will be using that? It can be:
```suggestion
    suspend fun sayHello()String {
        delay(2_000)
        return "Hello World"
    }
```